tools/gopackagesdriver

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+SCRIPT_DIR=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
+cd $SCRIPT_DIR/..
```

Comment:
Nit: this is a bit different from the default recommendation https://github.com/bazelbuild/rules_go/blob/master/tools/gopackagesdriver.sh

I think it's because you are planning to use the packages driver within another script, but it's unclear in the context of this PR why these are needed and how they would be used.

Reply:
```suggestion
# go-to-definition can take us to files outside the bazel workspace where bazel
# cannot be run, so cd back to the workspace root.
SCRIPT_DIR=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
cd $SCRIPT_DIR/..
```